Poker.BE/Poker.BE.Domain/Game/Hand.cs

```diff
@@ -25,6 +25,7 @@ public class Hand
         #endregion
 
         #region Properties
+		public bool handEnded { get; set;}
```

Comment:
@wainergal - please use `Ctrl+k Ctrl+d` - this keyboard shortcut will auto-format your code. don't deliver code that is not formatted - it's hard to understand & maintain. 👎